src/data/bucket/symbol_bucket.js

```diff
@@ -661,9 +736,25 @@ class SymbolBucket {
         }
     }
 
+    /**
+     * Add a single label & icon placement.
+     *
+     * Note that in the case of `symbol-placement: line`, the symbol instance's
+     * array of glyph 'quads' may include multiple copies of each glyph,
+     * corresponding to the different orientations it might take at different
+     * zoom levels as the text goes around bends in the line.
+     *
+     * As such, each glyph quad includes a minzoom and maxzoom at which it
+     * should be rendered.  This zoom range is calculated based on the 'layout'
+     * {text,icon} size -- i.e. text/icon-size at `z: tile.zoom + 1`. If the
+     * size is zoom-dependent, then the zoom range is adjusted at render time
+     * to account for the difference.
+     *
+     * @private
+     */
     addSymbolInstance(anchor, line, shapedTextOrientations, shapedIcon, layer, addToBuffers, collisionBoxArray, featureIndex, sourceLayerIndex, bucketIndex,
-        textBoxScale, textPadding, textAlongLine,
-        iconBoxScale, iconPadding, iconAlongLine, globalProperties, featureProperties) {
+        layoutTextSize, textBoxScale, textPadding, textAlongLine,
```

Comment:
Are `layoutTextSize` and `layoutIconSize` vestigial from some earlier version of this code? I don't see them being used.